Quiz_Question4_solution.ipynb

```diff
@@ -0,0 +1,93 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "98573d11",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "4ef3bbb1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create salary.csv file which contain sid,sname,salary.\n",
+    "#    Display the employee record whose name begins from \"S‟ also show no. of employee with first letter \"S‟ out of total record.\n",
+    "#          2,om,4000\n",
+    "#          5,sai,4200\n",
+    "#          Number of \"S‟ names are 2/5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "969e79c8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('c:\\\\sqlite3\\\\22bca35\\\\salary.csv','w',newline='') as f:\n",
+    "    w=csv.writer(f)\n",
+    "    h=['Id','Name','Salary']\n",
+    "    a=[[1,'om',20000],[1,'sai',21000],[3,'ram',10000],[4,'sudha',35000],[5,'krishna',11000]]\n",
+    "    w.writerow(h)\n",
+    "    w.writerows(a)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "d9159f66",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['1', 'sai', '21000']\n",
+      "['4', 'sudha', '35000']\n",
+      "Number of \"S‟ names are 2 / 5\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open('c:\\\\sqlite3\\\\22bca35\\\\salary.csv','r') as f:\n",
+    "    r=csv.reader(f)\n",
+    "    c=0\n",
+    "    t=0\n",
+    "    for i in r:\n",
+    "        if(i[1][0]=='s'):\n",
```

Comment:
what if i write the name started with capital letter than this program doesn't give correct output
can you please correct this problem ?

Reply:
sure ....
just add another condition using or in if statement with capital s
like this.....if(i[1][0]=='s' or i[1][0]=='S'):

Reply:
sorry i don't have any other option if you have let me know.

Reply:
You can use in (membership operator) instead of using "or"

Reply:
oh yes you are right .. thank you